kie-server-parent/kie-server-controller/kie-server-controller-client/src/main/java/org/kie/server/controller/client/rest/RestKieServerControllerClient.java

```diff
@@ -197,6 +197,12 @@ public ContainerList getContainers(ServerInstanceKey serverInstanceKey) {
         return throwUnsupportedException();
     }
 
+    @Override
+    public ContainerList getContainers(ServerTemplate serverTemplate,
+                                       ContainerSpec containerSpec) {
+        return throwUnsupportedException();
```

Comment:
@cristianonicolai the only question/concern is why we diverge in implementations of the clients - REST vs Websocket... do you believe REST client should not support it?

Reply:
@mswiderski I agree, we should support both clients, it is on my TODO to review the REST implementation that was left behind.